spec/api-app-spec.js

```diff
@@ -112,6 +112,25 @@ describe('app module', () => {
     })
   })
 
+  describe('app.setLocale()', () => {
```

Comment:
The PR fixes the use of `--lang` flag, maybe the test should be moved to `chromium-spec.js` and describe the flag.